acceptance_tests/features/who.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const translation = require('./../../apps/complaints/translations/en/default');
+
+Feature('I am on the /who step');
+
+Scenario('The page loads and I am on the Personal and contact details page', (I) => {
```

Comment:
I believe this is a section, not a page, this will be true for any page in this section